app/config/local/app.php

```diff
@@ -26,7 +26,7 @@
 	|
 	*/
 
-	'url' => 'http://localhost/atlas',
+	'url' => 'http://openmrsatlas.local',
```

Comment:
Unless you have a reason for changing it, I would leave it unchanged. You do not need to create a new pull request; rather, you can simply [update this pull request](http://stackoverflow.com/questions/7947322/preferred-github-workflow-for-updating-a-pull-request-after-code-review). :smile: